forge-gui/res/cardsfolder/upcoming/roar_of_resistance.txt

```diff
@@ -0,0 +1,7 @@
+Name:Roar of Resistance
+ManaCost:1 R
+Types:Enchantment
+S:Mode$ Continuous | Affected$ Creature | AddKeyword$ Menace | Description$ Creatures you control have menace.
+T:Mode$ AttackersDeclared | AttackingPlayer$ Player | AttackedTarget$ Player,Planeswalker| ValidAttackers$ Creature | Execute$ TrigPumpAll | TriggerDescription$ Whenever one or more creatures attack, you may pay {1}{R). If you do, creatures attacking your opponents and/or planeswalkers they control get +2/+0 until end of turn.
```

Comment:
There's really no reason to include these non-limiting AttackingPlayer + AttackedTarge + ValidAttackers